### PR TITLE
Update Register your app topic for OKTA-378784

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?**  Update *Register your app* topic, specifically the CLI/Website instructions for *Tell Okta about the app*
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-378784](https://oktainc.atlassian.net/browse/OKTA-378784)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
+You can use either the Okta CLI or the Admin Console to register your app to your Okta org for user authentication. After your app is registered, your Okta org generates the issuer ID, client ID, and client secret for you to integrate into your app build.
 
-For simplicity, we'll assume that your app is a server-side web app, rather than one of the other supported types of apps, and that you want to use the Okta-hosted sign-in page, rather than host the sign-in page yourself.
+The following instructions show you how to register a server-side web app to have an Okta-hosted sign-in page. For Admin Console instructions on how to register all the supported app types and sign-in methods, see [Create a new Okta app integration](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_App_Integration_Wizard.htm).
 
 #### Start from a sample app
 
-To start, find the example applications provided by Okta on GitHub for the language or framework you work in:
+You can review and build Otka's sample applications from GitHub to learn how the various languages or frameworks can be used for Okta authentication.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md:11]:
```suggestion
You can review and build Okta's sample applications from GitHub to learn how the various languages or frameworks can be used for Okta authentication.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+
+Find the sample applications that matches your app's language or framework:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md:13]:
```suggestion
Find the sample applications that match your app's language or framework:
```

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

**Review by jakubvul-okta — APPROVED** [review]

**Review by gormanho-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
 1. Enter the command: `okta apps create`.
-2. Give your app a name.
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
-4. Select **Other**.
-5. For the **Login redirect URI**, enter: `http://localhost:8080/login/callback`.
+2. Type your app name.
+3. Select **Single Page App** or **Web** as the **Type of Application**.
+    - **Single Page App**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
+    - **Web**&mdash;for applications using tokens solely in server-side code
+
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:8]: There is a missing step here when creating a "Web" app using the cli:

```txt
Type of Application
> 1: Okta Spring Boot Starter
> 2: Spring Boot
> 3: JHipster
> 4: Quarkus
> 5: Other
Enter your choice [Other]: 
```

Using the default creates a `.okta.env` file in the current directory:

```sh
export OKTA_OAUTH2_ISSUER="https://<okta-url>/oauth2/default"
export OKTA_OAUTH2_CLIENT_ID="<client-id>"
export OKTA_OAUTH2_CLIENT_SECRET="<secret-is-here>"
```

The other "web" options will write the configuration directly to an application-specific location (for example Spring Boot is typical `src/main/resources/application.properties`

These values are NOT shown to the screen (they contain a secret), instead the user sees something like:

```txt
Okta application configuration has been written to: <current-directory-here>/.okta.env
```

SPA applications don't lend themselves well to this type of configuration (which is another reason why those values are just displayed on the screen)

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
 
 1. Enter the command: `okta apps create`.
-2. Give your app a name.
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
-4. Select **Other**.
-5. For the **Login redirect URI**, enter: `http://localhost:8080/login/callback`.
+2. Type your app name.
+3. Select **Single Page App** or **Web** as the **Type of Application**.
+    - **Single Page App**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
+    - **Web**&mdash;for applications using tokens solely in server-side code
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:7]: "solely" might be too strong of a word here, it's possible you would send a token to other applications.

Users also might not understand what access tokens are at this point (ideally they shouldn't need to), but this is a bit outside the scope of this PR

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md [hunk 5]
```diff
@@ -4,18 +4,29 @@ In the Admin Console, complete these steps:
 
 2. Click **Add Application**.
 
-3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code. Pick **Web** if your application is using the tokens solely in server-side code.
+3. From the **Add Application** page, click **Create New App**.
 
-4. Enter the **Name** of your app.
+4. Select a **Platform**:
+    - **Single Page App(SPA)**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md:10]:
```suggestion
    - **Single Page App (SPA)**&mdash;for applications using tokens in Angular, React, Vue, or other browser-side code
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:21]: Thanks for the feedback! I've added these steps and notes to the CLI instructions.

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md (hunk 4 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md]: I've re-worded this description. Please review again. Thanks!

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md (hunk 5 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md]: Thanks Brian. I've reorganized this section and added the space.

**Comment by mraible** [comment]
FWIW, it seems this pull request I just created changes similar files. https://github.com/okta/okta-developer-docs/pull/2020

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 6]
```diff
@@ -1,59 +1,30 @@
...
-Enter your Post Logout Redirect URI(s) [http://localhost:8080/]:
-```
 When the Okta CLI finishes adding your app, the issuer ID, client ID, and client secret credentials are written to your app-specific configuration file.
 ```
 Configuring a new OIDC Application, almost done:
-Created OIDC application, client-id: 0oad95td4fs8kyPQc5d6
+Created OIDC application, client-id: <client-ID>
 
-Okta application configuration has been written to: /Users/<currentUser>/.okta.env
+Okta application configuration has been written to: <home-dir>/.okta.env
```
Comment (mraible) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:22]: I wouldn't use `<home-dir>` here because it doesn't put the file in their home directory. I'd use `<app-dir>` or `/path/to/app/.okta.env`. The latter is what we use on the developer blog.

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:22]: ok, thanks!
